### DISCUSSION TITLE
Behaviour when project has been removed

### DISCUSSION THREAD

**Discussion by raul3z** [getsentry/sentry-python#3333] [Q&A] [answered]
Hi there!

I'd like to ask what happens when a project is removed but the code still points to it, I was making some tests with a testing project and it seems do nothing (in fact, the envelope is sent as normal but never reach the server because the project does not exist), I just wanted to be sure about this behaviour.

Thanks in advance.

**Comment by antonpirker** [comment, marked as answer]
I think if you send envelopes for a project, that does not exist anymore, we just drop them.

**Reply by raul3z** [reply]
Thank you!
